### ISSUE TITLE
Unification of c16 and plus4

### ISSUE THREAD

**Issue by polluks** [cc65/cc65#612]
Merge both targets like atari and atarixl
````
Tm Sz Pr Fn Del Name
-- -- -- -- --- -------
    x  x        break.s
    x  x        cgetc.s
    x  x        clrscr.s
    x  x        crt0.s
    x  x        devnum.s
    x  x        get_tv.s
    x  x        irq.s
    x  x        joy_stat_stddrv.s
    x  x        joy_stddrv.s
    x  x        kbhit.s
    x  x        kclose.s
    x  x        libref.s
    x  x        mainargs.s
    x  x        randomize.s
    x  x        sysuname.s
    x  x        _scrsize.s
       x        color.s
       x        conio.s
       x        cputc.s
       x        revers.s
       x        status.s
       x        systime.s
              x kernal.s
````

**Comment by oliverschmidt** [comment]
This seems (of course) a vrey good idea. However, I have two questions:
* What is the meaning of the column headers `Tm`, `Sz`, `Pr`, `Fn` and `Del`?
* Why is this an issue and no pull request? ;-)

**Comment by polluks** [comment]
This was your idea IIRC :-)
The table is just a reminder which file differs somehow...

**Comment by spiro-trikaliotis** [comment]
It might have beeen Oliver's idea, but it does not explain the meaning of the columns.

**Comment by polluks** [comment]
@spiro-trikaliotis C'mon, it's Size, Protection and Deleted. So six files are identical, nothing fancy.

**Comment by groessler** [comment]
@polluks: Most of your messages appear to be some kind of an intelligence test to me. Sadly, I'm not that bright. Is it so difficult or an inconvenience to really write out what you want to say?

Your explanation is incomplete.

Size (I guess that's "Sz"): what does 'x' mean? Different? Or same?
Protection ("Pr"?): what is that? 
Deleted ("Del"?): what is that?

And you haven't explained the "Tm" and "Fm" columns.

**Comment by polluks** [comment]
Indeed, I could save the output locally but using this thread seemed to be handy.
'x' means different, by the way the table is an lha hunt, see section 2.4.7 of
https://raw.githubusercontent.com/theflyingape/rpgbbs/master/LhA.man
"Tm" and "Fn" are simply not relevant.

**Comment by oliverschmidt** [comment]
> Most of your messages appear to be some kind of an intelligence test to me.

;-)

**Comment by groessler** [comment]
Isn't this more or less the same as #461?

**Comment by groessler** [comment]
Reopening that...

**Comment by polluks** [comment]
> Reopening that...

Completed by #2919
